test/unit/test_events.py

```diff
@@ -45,3 +46,32 @@ def test_formatting(self):
         # were no args passed after the initial message
         event = AdapterEventDebug(name="dbt_tests", base_msg="boop{x}boop", args=())
         self.assertTrue("boop{x}boop" in event.message())
+
+class TestEventCodes(TestCase):
+
+    def get_all_subclasses(self,cls):
```

Comment:
minor formatting thing
```suggestion
    def get_all_subclasses(self, cls):
```